CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ look at [README.md](README.md) to understand how Go modules impact build and dev
 
 FEATURES:
 
-* New insert and eject media functions 
+* New insert and eject media functions
+* Added metadata add/remove functions to vm
 
 IMPROVEMENTS:
+
+* vApp metadata now is attached to the vapp rather to first vm in vapp.
```

Comment:
If you can keep the vApp name pattern (I have to get used to it myself) :)

Reply:
Fixed